sklearn/tests/test_metaestimators_metadata_routing.py

```diff
@@ -159,10 +159,6 @@ def predict_log_proba(self, X, sample_weight="default", metadata="default"):
         "X": X,
         "y": y_multi,
         "routing_methods": ["fit", "partial_fit"],
-        "warns_on": {
-            "fit": ["sample_weight", "metadata"],
-            "partial_fit": ["sample_weight"],
-        },
```

Comment:
removing these and corresponding tests since we don't deal with backward compatibility now.